tests/virtctl/scp.go

```diff
@@ -23,6 +23,48 @@ var _ = Describe("[sig-compute][virtctl]SCP", func() {
 	var pub ssh.PublicKey
 	var keyFile string
 	var virtClient kubecli.KubevirtClient
+
+	copyNative := func(src, dst string, recursive bool) {
+		args := []string{
+			"scp",
+			"--namespace", util.NamespaceTestDefault,
+			"--username", "cirros",
+			"--identity-file", keyFile,
+			"--known-hosts=",
+		}
+		if recursive {
+			args = append(args, "--recursive")
+		}
+		args = append(args, src, dst)
+
+		Expect(clientcmd.NewRepeatableVirtctlCommand(args...)()).To(Succeed())
+	}
+
+	copyLocal := func(src, dst string, recursive bool) {
+		args := []string{
+			"scp",
+			"--local-ssh=true",
+			"--namespace", util.NamespaceTestDefault,
+			"--username", "cirros",
+			"--identity-file", keyFile,
+			"-t", "-o StrictHostKeyChecking=no",
+			"-t", "-o UserKnownHostsFile=/dev/null",
+			"-t", "-O",
+		}
+		if recursive {
+			args = append(args, "--recursive")
+		}
+		args = append(args, src, dst)
+
+		_, cmd, err := clientcmd.CreateCommandWithNS(util.NamespaceTestDefault, "virtctl", args...)
+		Expect(err).ToNot(HaveOccurred())
+		cmd.Env = append(cmd.Env,
```

Comment:
Is this required if we've already `Expect(os.Setenv("SSH_AUTH_SOCK", "/dev/null")).To(Succeed())` in `BeforeEach`?

Reply:
You are right, it is not required again.